win32/test/test_win32gui.py

```diff
@@ -61,5 +64,123 @@ def test_memory_not_writable(self):
         self.assertRaises(TypeError, operator.setitem, got, 0, 1)
 
 
+class TestEnumWindowsFamily(unittest.TestCase):
+    @classmethod
+    def enum_callback_sle(cls, handle, data):
+        win32api.SetLastError(1)
+        return data
+
+    @classmethod
+    def enum_callback_exc(cls, handle, data):
+        raise ValueError()
+
+    @classmethod
+    def enum_callback(cls, handle, data):
+        return data
+
+    def setUp(self):
+        self.default_data_set = (None, -1, 0, 1, True, False)
+        self.type_data_set = (2.718282, "", (), {})
+
+    def test_enumwindows(self):
+        win32api.SetLastError(0)
+        for data in (0, False):
+            self.assertRaises(
+                pywintypes.error, win32gui.EnumWindows, self.enum_callback_sle, data
+            )
+        for data in (None, 1, True):
+            self.assertIsNone(win32gui.EnumWindows(self.enum_callback_sle, data))
+        win32api.SetLastError(0)
+        for data in self.default_data_set:
+            self.assertIsNone(win32gui.EnumWindows(self.enum_callback, data))
+        for data in self.default_data_set:
+            self.assertRaises(
+                ValueError, win32gui.EnumWindows, self.enum_callback_exc, data
+            )
+        if sys.version_info[:2] >= (3, 10):
```

Comment:
Slicing is redundant here

```suggestion
        if sys.version_info >= (3, 10):
```

Reply:
done